<launch>
    <!-- <node pkg="course_agv_nav" type="global_planner.py" name="global_planner" ns="course_agv"  output="screen"/> -->
    <!-- <node pkg="course_agv_nav" type="local_planner.py" name="local_planner" ns="course_agv" output="screen"/> --> -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find course_agv_nav)/launch/nav_real.rviz"/>
	<node pkg="course_agv_nav" type="tf" name="my_tf"/>
	<node pkg="course_agv_nav" type="icp_ekf.py" name="localization">
		<param name="range_threshold" value="0.5"/>
		<param name="radius_max_th" value="0.4"/>
		<param name="landMark_min_pt" value="2"/>
		<param name="min_match" value="2"/>
		<param name="max_iter" value="10"/>
		<param name="tolerance" value="0"/>
		<param name="dis_th" value="0.05"/>
	</node>
    <node pkg="course_agv_nav" type="global_planner.py" name="global_planner" ns="course_agv"  output="screen"/> 
    <node pkg="course_agv_nav" type="local_planner.py" name="local_planner" ns="course_agv" output="screen"/>
</launch>